pilot/helpers/agents/Developer.py

```diff
@@ -239,8 +240,19 @@ def step_test(self, convo, test_command):
 
     def get_run_command(self, convo):
         llm_response = convo.send_message('development/get_run_command.prompt', {}, COMMAND_TO_RUN)
-        self.run_command = llm_response['command'].strip('`')
-        self.run_command = self.run_command.strip('\n')
+        self.run_command = llm_response['command']
+
+        # Pattern for triple backtick code block with optional language
+        triple_backtick_pattern = r"```(?:\w+\n)?(.*?)```"
+        triple_match = re.search(triple_backtick_pattern, self.run_command, re.DOTALL)
+        # Pattern for single backtick
+        single_backtick_pattern = r"`(.*?)`"
+        single_match = re.search(single_backtick_pattern, self.run_command, re.DOTALL)
+
+        if triple_match:
+            self.run_command = triple_match.group(1).strip()
+        elif single_match:
+            self.run_command = single_match.group(1).strip()
```

Comment:
it happened to me (very rarely) that i would get response from LLM '``\ncmd\n``'. Notice that there are only 2 ` and not 1 or 3.

This current implementation would fail for example above and give empty string as result. Why was this change done?

Reply:
The previous implementation would just remove 1 ` from each side. I would say that this is the reason why you would get 2 ` - the implementation in this PR is because LLMs usually return either a single line code (with 1 `) or multi line code (with 3 `)